transformations/summarization_transformation/README.md

```diff
@@ -0,0 +1,32 @@
+# Sentence Summarizaiton 🦎  + ⌨️ → 🐍
+This perturbation extracts all types of text sources (sentence, paragraph, etc.) for the summarization of the sentence without changing meanings.
+
+Author name: Jing Zhang (jing.zhang2@emory.edu, Emory University)
+
+Author name: Zhexiong Liu (zhexiong@cs.pitt.edu, University of Pittsburgh)
+
+## What type of a transformation is this?
+This transformation acts like extracting subjects, verbs, and objects as an summarization of the given sentence. Keep the original basic meanings and retain the negations.
+
+It also robust to generate the summarizaiton of the sentence which contains grammar issues (test case 2).
+
+## What tasks does it intend to benefit?
+It is useful for question & answering, sentiment analysis, sentence pair relatedness, and other tasks. No need to require any pretrained model to abstract the sentence, and no need to require entire dataset to generate 
+summarization of the sentence (such as TF-IDF method), it summarizes only based on each individual input sentence.
```

Comment:
Could you add some more details / insights into the main logic of how the transformation works?
While it might not be using any pretrained models or datasets, a brief explanation of how it works and what resources it does use will be helpful?

Reply:
Thanks for your comments! The basic idea is to extract a triplet (subject, predicate, object) from a given sentence, which contains salient information of the sentence. We used POS tags to label each token and used our algorithm to extract the triplet based on the tags.

Reply:
This paper (https://aclanthology.org/2021.acl-long.70.pdf) demonstrates that nouns, verbs are the most useful information in the given context, so our triplet extraction shows great potential for information extraction with the ability to keep most salient information in a triplet.

Reply:
Adding these details in a section in the readme would help potential users.